Makefile

```diff
@@ -291,37 +291,39 @@ SCHEMAS_DEPLOY_NPM = echo "//registry.npmjs.org/:_authToken=${NPM_TOKEN}" > .npm
 SCHEMAS_VERSION_PYPI = poetry version ${SCHEMAS_VERSION};
 SCHEMAS_VERSION_NPM = npm version --allow-same-version ${SCHEMAS_VERSION};
 
-schemas_build:  ## Build schemas
+schemas_docker_build:  ## Build schemas docker image
```

Comment:
Do we need to update the README anywhere to refer to these changes?  Does this change the release process?

Reply:
I checked and couldn't find any reason this would be a problem. The idea here is to just clarify what is actually happening and bring the Makefile and README into sync:

The README (along with an error message in the Makefile) was actually kind of wrong, in that it was saying to run `schemas_build` to update the typescript types, but that wasn't happening until this change. The CI config doesn't use `schemas_build` so shouldn't be a problem there either. And I replaced all of the spots where `schemas_build` was a prereq to another make rule with `schemas_docker_build`.